### ISSUE TITLE
the datasets are too big to be downloaded

### ISSUE THREAD

**Issue by ChChwang** [peteanderson80/bottom-up-attention#49]
the datasets are too big to be downloaded

**Comment by supriya-pandhre** [comment]
Yes, I am also facing issue in downloading dataset due to large size of a single file.
@peteanderson80 would it be possible to provide the dataset in parts?

**Comment by ManasiPat** [comment]
I am not been able to download the datasets either. Can they be please put in parts.

**Comment by peteanderson80** [comment]
There is an alternative download option now using Google cloud storage. It is user pays but it is very fast and should work much better, even with everything in a single file.
